ui/admin/app/components/form/credential-store/vault/index.hbs

```diff
@@ -220,47 +219,49 @@
     {{/if}}
   </form.checkbox>
 
-  <form.textarea
-    @name='client_certificate'
-    @type='text'
+  <Hds::Form::Textarea::Field
```

Comment:
Any reason we didn't mark any of these text areas as optional? I believe these are all optional